runtime/lua/vim/lsp/diagnostic.lua

```diff
@@ -159,44 +159,45 @@ local function diagnostic_vim_to_lsp(diagnostics)
   end, diagnostics)
 end
 
----@type table<integer,integer>
+---@type table<integer, integer>
```

Comment:
Just as a heads up, I've seen cases where `lua2dox` chokes on spaces in type definitions. Example: https://github.com/neovim/neovim/pull/25136/commits/02734df2f9ad329ee2bb371f8ce632bfd7cd2b65

I don't think it will be an issue in this particular case (the issue in the commit above was because the type was optional). This should probably be fixed in `lua2dox` at some point.